src/invoices/InvoiceDetails/InvoiceDetails.js

```diff
@@ -1,18 +1,52 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
```

Comment:
@NikitaSedyx what do you think of `import get from 'lodash/get';`? I think it might reduce bundle size

Reply:
it's best practice, but not on our project :) this won't decrease bundle because of other modules

Reply:
of others - yes, but in local dev setup it might impact our experience in a good way, isn't it?

Reply:
several KBs of memory for dev, don't think it's a problem